cluster/v1/0000_00_clusters.open-cluster-management.io_managedclusters.crd.yaml

```diff
@@ -34,10 +34,10 @@ spec:
       description: "ManagedCluster represents the desired state and current status
         of managed cluster. ManagedCluster is a cluster scoped resource. The name
         is the cluster UID. \n The cluster join process follows a double opt-in process:
-        \n 1. agent on managed cluster creates CSR on hub with cluster UID and agent
-        name. 2. agent on managed cluster creates ManagedCluster on hub. 3. cluster
+        \n 1. Agent on managed cluster creates CSR on hub with cluster UID and agent
+        name. 2. Agent on managed cluster creates ManagedCluster on hub. 3. Cluster
```

Comment:
Remove possessive here by replacing:
"Cluster admin on hub approves the CSR for the ManagedCluster's UID and agent name."
with 
"Cluster admin on the hub approves the CSR for the UID and agent name of the ManagedCluster."

Reply:
done

Reply:
fixed